src/libcore/lib.rs

```diff
@@ -208,3 +208,87 @@ pub use coresimd::simd;
 #[unstable(feature = "stdsimd", issue = "48556")]
 #[cfg(not(stage0))]
 pub use coresimd::arch;
+
+// FIXME: move to appropriate location
```

Comment:
Please move this to libcore/asserting.rs (by analogy with runtime support for panicking in libcore/panicking.rs).